ui/pages/destinations/index.js

```diff
@@ -221,6 +221,14 @@ function Details({ destination, onDelete }) {
                 : '-'}
             </div>
           </div>
+          <div className='flex flex-row items-center'>
+            <div className='w-1/3 text-2xs text-gray-400'> Version</div>
```

Comment:
This is awesome. Suggestion: `Connector Version` so users don't confuse it with Kubernetes versions (especially once we get to 1.x)!

Reply:
I was under the impression this panel is destination metadata. In that case, it should be fairly obvious this is the destination version as nothing else in this view references the actual Kubernetes cluster

Reply:
I think "connector-version" is easier to understand too. There's a lot of versions in the mix so being more specific can't hurt.